static-assets/components/cstudio-common/resources/de/base.js

```
@@ -639,103 +639,103 @@ CStudioAuthoring.Messages.registerBundle('browse', 'de', {
+  audit: 'Protokoll',
+  groups: 'Gruppen',
+  contentTypes: 'Inhaltstypen',
+  configuration: 'Konfiguration',
+  workflowJobs: 'Workflow Jobs',
+  workflowStates: 'Workflow Status',
+  loggingLevels: 'Log-Stufen',
+  logConsole: 'Log-Konsole',
+  previewsync: 'Vorschau Sync',
+  syncfromrepository: 'Sync aus Repository',
+  deploymentQueues: 'Deployment Queue',
+  label: 'Inhaltstypen',
```

Biome (2.4.13)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)